metadata-db-app/src/cmr/metadata_db/data/oracle/search.clj

```diff
@@ -192,6 +218,44 @@
        (when-let [start-index (find-batch-starting-id db table params)]
          (lazy-find (max requested-start-index start-index)))))))
 
+(defn find-concepts-in-batches-with-stmt
```

Comment:
I recognize this code from elsewhere, seems like a copy/paste with a tiny bit of change. Let's see if there's an alternative.

Reply:
As we have discussed, we will work the refactoring issue (CMR-5704) later.